code-challenges.test.js

```diff
@@ -24,25 +29,90 @@
 
 // b) Create the function that makes the test pass.
 
+//create fib variable
+//for fib define num values 
+//for loop states that the function will act on numbers greater than 2, but less than or equal to the example inputs (6,10). Function will iterate through the given parameters
+//return array of 6 and 10 in fib.
+
+
```

Comment:
Basic structure of a function. 
const fib () => {return "output"}
The goal of the function is to allow you to dynamically take in any input, use the desired methods to produce the expected output, and return that item as the actual output of your function.